pybloqs/__init__.py

```diff
@@ -9,29 +9,6 @@
 from pybloqs.block.wrap import Box, Paragraph
 from pybloqs import plot as bxpl
 
-from six import StringIO
-
-
-def interactive(verbose=True):
```

Comment:
Do you need to import this now that it's moved? Either that or remove it from `__all__` here on L35